src/databricks/labs/lsql/cli.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+
+from databricks.labs.blueprint.cli import App
+from databricks.labs.blueprint.entrypoint import get_logger
+from databricks.sdk import WorkspaceClient
+
+from databricks.labs.lsql.dashboards import Dashboards
+
+logger = get_logger(__name__)
+lsql = App(__file__)
+
+
+@lsql.command
+def create_dashboard(w: WorkspaceClient, folder: Path = Path.cwd()):
+    """Create a dashboard from queries"""
+    logger.info("Creating dashboard ...")
+    dashboards = Dashboards(w)
+    lakeview_dashboard = dashboards.create_dashboard(Path(folder))
+    sdk_dashboard = dashboards.deploy_dashboard(lakeview_dashboard)
```

Comment:
does it open deployed dashboard or dashboard-in-progress?

Reply:
it opens the unpublished dashboard

Reply:
see screenshot above